units/util_test.go

```diff
@@ -7,12 +7,20 @@ import (
 	"github.com/evergreen-ci/evergreen"
 	"github.com/evergreen-ci/evergreen/db"
 	"github.com/evergreen-ci/evergreen/model/host"
+	"github.com/evergreen-ci/evergreen/model/task"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestHandlePoisonedHost(t *testing.T) {
 	assert := assert.New(t)
-	assert.NoError(db.ClearCollections(host.Collection))
+	assert.NoError(db.ClearCollections(host.Collection, task.Collection))
```

Comment:
Can you add tests for other non-happy cases (e.g. the host is already decommissioned/quarantined/terminated, it's not currently running a task, it's a docker container)?

Reply:
done